lib/helper.js

```diff
@@ -292,6 +321,15 @@ Helper.prototype.waitForModalTransition = function() {
   }, self.MAX_TIMEOUT, 'should wait for modal to transition');
 };
 
+Helper.prototype.waitForModalToClose = function(modal) {
+  var self = this;
+  browser.wait(function() {
+    return modal.isPresent().then(function(res) {
+      return !res;
+    });
+  }, self.MAX_TIMEOUT, 'should wait for modal to close');
+};
```

Comment:
This function seems like it should be generalized to work for any element ... and it looks like it's just checking to see if an element is not present.